redis-hbo-provider/src/main/java/com/facebook/presto/statistic/RedisProviderSerdeException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.statistic;
+
+import com.facebook.presto.spi.PrestoException;
+
+import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
+
+public class RedisProviderSerdeException
+        extends PrestoException
+{
+    public RedisProviderSerdeException(String message)
+    {
+        this(message, null);
+    }
+
+    public RedisProviderSerdeException(String message, Throwable cause)
+    {
+        super(GENERIC_INTERNAL_ERROR, "RedisProviderPlugin Exception: " + message, cause);
```

Comment:
This seems worth having its own error code.  If the exceptions are specific to the Redis HBO provider, then maybe a separate error code provider/error code for this.  Otherwise, maybe an HBO failure error code?

Reply:
Its just one error code for now so will create a new HBO_FAILURE, this can be used in other places as well in future.